jobs/JOBSPROC_RAP_DUMP_POST

```diff
@@ -132,7 +134,7 @@ export SENDDBN=${SENDDBN:-YES}
 # Paths for obsproc_network items
 # -------------------------------
 export HOMEobsproc=${HOMEobsproc:\
--${NWROOT:?}/obsproc.${obsproc_ver:?\
+-${PACKAGEROOT:?}/obsproc.${obsproc_ver:?\
```

Comment:
See same as JOBSPROC_RAP_DUMP. Also not missing closed "}". I added it in my example.
export HOMEobsproc=${HOMEobsproc:\
-${PACKAGEROOT:?}/${obsNET}/${obsproc_ver}:?\